resources/html/test/testPlayground.html

```diff
@@ -0,0 +1,188 @@
+<!--
+    This file is a html page with various element used for testing candybean's
+    functionalities
+-->
+
+<html>
+    <head>
+        <title>Candybean Test Page</title>
+        <script type="text/javascript">
```

Comment:
In [onOffScreen.html](onOffScreen.html), the JS and CSS portion of the code are intended. I believe that the file will be a bit easier to read with some indentation as well, so let's keep things consistent.